fuse_models/src/unicycle_2d.cpp

```diff
@@ -204,6 +204,7 @@ void Unicycle2D::onInit()
   process_noise_covariance_ = fuse_core::Vector8d(process_noise_diagonal.data()).asDiagonal();
 
   private_node_handle_.param("scale_process_noise", scale_process_noise_, scale_process_noise_);
+
```

Comment:
I think the extra lines for the others are only because they need extra work other than just calling `param`, so I wouldn't add a new line here:
```suggestion
```